### PR TITLE
build: update twoliter to v0.2.0

### PR DESCRIPTION
**Pull request by sumukhballal**
**Issue number:**

Closes #

**Description of changes:**

Updates twoliter to version v0.2.0 and Twoliter.toml to new specification.

Reference: https://github.com/bottlerocket-os/twoliter/issues/207

**Testing done:**
- [x] Build a variant of Bottlerocket (aws-k8s-1.27 & aws-ecs-1).
- [x] Built an aws-ecs-1 AMI, and made sure Bottlerocket boots.

**Terms of contribution:**

By submitting this pull request, I agree that this contribution is dual-licensed under the terms of both the Apache License, version 2.0, and the MIT license.

### CODE REVIEW

**Comment by sumukhballal** [comment]
Updated Twoliter version to v0.2.0-rc2 and added @webern 's [commit](https://github.com/bottlerocket-os/bottlerocket/pull/3962/commits/c7f1177f79d8ac3ac44d6228c1115f8a3997f213
)

**Review by jpculp — APPROVED** [review]

**Comment by jpculp** [comment]
Using `v0.2.0-rc3` I was able to build, boot, and repack `aws-k8s-1.28`.

**Comment by jpculp** [comment]
Also built and booted `aws-ecs-2`.

File: Makefile.toml [hunk 1]
```diff
@@ -7,7 +7,7 @@ BUILDSYS_ROOT_DIR = "${CARGO_MAKE_WORKING_DIRECTORY}"
 # For binary installation, this should be a released version (prefixed with a v,
 # for example v0.1.0). For the git sourcecode installation method, this can be
 # any git rev, e.g. a tag, sha, or branch name.
-TWOLITER_VERSION = "v0.1.1"
+TWOLITER_VERSION = "v0.2.0-rc3"
```
Comment (webern) [inline Makefile.toml]:
```suggestion
TWOLITER_VERSION = "v0.2.0"
```

**Review by webern — CHANGES_REQUESTED** [review]
Looks good! Blocking for the final Twoliter tag only.

**Comment by sumukhballal** [comment]
Updated twoliter version to v0.2.0

**Review by webern — APPROVED** [review]

**Pull request merged**